.gitignore

```diff
@@ -140,3 +140,6 @@ dmypy.json
 # stagedfright 
 
 *.stagedfright-allowed
+
+# Gurobi Troubleshooting files
+gurobi.*
```

Comment:
I'm not sure how relevant this can be, but this pattern would also exclude a possible future `gurobi.py`.